botogram/objects/mixins.py

```diff
@@ -184,6 +184,15 @@ def send_contact(self, phone, first_name, last_name=None, *, reply_to=None,
 
         return self._api.call("sendContact", args, expect=_objects().Message)
 
+    @_require_api
+    def delete_message(self, message):
+        if hasattr(message, "message_id"):
+            message = message.message_id
+
+        return self._api.call("deleteMessage", {
+            "chat_id": self.id, "message_id": message
```

Comment:
Can you put all of these on their own line, with a traling comma in all of them?

```python
{
    "chat_id": self.id,
    "message_id": message,
}
```